src/commands/history/mod.rs

```diff
@@ -0,0 +1,100 @@
+use clap::Args;
+use sqlite::{Connection, State, Value};
+use tabled::{
+    settings::{peaker::PriorityMax, Width},
+    Table, Tabled,
+};
+
+#[derive(Debug, Args)]
+#[clap(name = "history", about = "Display the previous search history")]
+pub struct History {
+    #[clap(short, long, default_value = "10")]
+    limit: i64,
```

Comment:
How about support [`Offset/limit pagination`](https://planetscale.com/blog/mysql-pagination#offset-limit-pagination)?

Providing only the `limit` value as an option might make it difficult for users to view past records.